bundle/manifests/nodeobservability.olm.openshift.io_nodeobservabilityruns.yaml

```diff
@@ -49,6 +49,14 @@ spec:
                 required:
                 - name
                 type: object
+              restoreMCOStateAfterRun:
+                description: RestoreMCOAfterRun - flag used to restore previous machineconfig
+                  state after run
+                type: boolean
+              runType:
+                description: RunType identifies the runtype This setting will allow
+                  for logic to bypass machine config settings
```

Comment:
Do you think we can also add a list of accepted values in the description for the end user?

Reply:
Yes added